plugins/reindex/src/main/java/org/elasticsearch/plugin/reindex/AbstractBulkByScrollRequest.java

```diff
@@ -72,6 +74,17 @@
      */
     private WriteConsistencyLevel consistency = WriteConsistencyLevel.DEFAULT;
 
+    /**
+     * Initial delay after a rejection before retrying request. Defaults to 50ms. With the default maxRetries to total time to retry
+     * rejects is about 25 seconds per rejection.
```

Comment:
> "With the default maxRetries to total time to retry rejects is about 25 seconds per rejection."

This is a little unclear but I think you're saying if we go through all 10 retries with this default initial backoff it'll be around 25 seconds before it's finally failed?